sweetalert2.d.ts

```diff
@@ -337,6 +337,10 @@ declare module 'sweetalert2' {
     const version: string
   }
 
+  interface Promisify<T> {
+    readonly toPromise: () => T;
```

Comment:
We don't require the `toPromise` member to be `readonly` do we?

Reply:
In fact, `readonly` for this case doesn't matter at all.